Lib/fontTools/varLib/__init__.py

```diff
@@ -313,17 +313,18 @@ def _merge_OTL(font, model, master_fonts, axisTags, base_idx):
 	print("Merging OpenType Layout tables")
 	merger = VariationMerger(model, axisTags, font)
 
-	merge_tables(font, merger, master_fonts, axisTags, base_idx, ['GPOS'])
-	store = merger.store_builder.finish()
-	try:
-		GDEF = font['GDEF'].table
-		assert GDEF.Version <= 0x00010002
-	except KeyError:
-		font['GDEF']= newTable('GDEF')
-		GDEFTable = font["GDEF"] = newTable('GDEF')
-		GDEF = GDEFTable.table = ot.GDEF()
-	GDEF.Version = 0x00010003
-	GDEF.VarStore = store
+	if 'GPOS' in font:
```

Comment:
Yes, it is the same issue as #699. I am still somewhat new to github. Once I have submitted a pull request, can I back out specific changes?